app/models/skill_with_rating.rb

```diff
@@ -1,18 +1,29 @@
 class SkillWithRating
-  attr_reader :skill, :rating
+  attr_reader :skill, :rating, :user
 
   def initialize(skill, user)
     @skill = skill
     @rating = skill.ratings.where(user: user).first
+    @user = user
   end
 
   def attributes
     skill.attributes.tap do |skill|
-      skill["rating"] = rating.id
+      skill["featured"] = featured_skills.include? skill if skills_present?
+      skill["rating"]   = rating.score if rating
+      # skill["skill_id"] = skill["id"]
     end
   end
 
-  def to_json
-    attributes.to_json
+  def skills_present?
+    user.cohort && user.cohort.skills
+  end
+
+  def featured_skills
+    user.cohort.skills.map(&:attributes)
+  end
+
+  def as_json(options={})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.